support-modules/acquisition-events/build.sbt

```diff
@@ -9,6 +9,8 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-core" % circeVersion,
   "io.circe" %% "circe-generic" % circeVersion,
   "com.amazonaws" % "aws-java-sdk-kinesis" % "1.12.10",
+  "com.gu" %% "acquisitions-value-calculator-client" % "2.0.6",
+  "com.squareup.okhttp3" % "okhttp" % okhttpVersion,
```

Comment:
these deps were coming via support-models/acquisition-event-producer

Reply:
there is an sbt plugin that flags up when you rely on transitive dependencies, although it's probably not worth using for us

Reply:
I think we should get rid of the code which retrieves AV for GA. We never really use it in our tests and it would allow us to drop these dependencies.